tensorflow_io_plugin_gs/core/file_system_plugin_gs.cc

```diff
@@ -0,0 +1,36 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include "tensorflow_io_plugin_gs/core/file_system_plugin_gs.h"
+
+#include "absl/strings/ascii.h"
+
+void TF_InitPlugin(TF_FilesystemPluginInfo* info) {
```

Comment:
is this file copied from somewhere? if so can we document the source ?

Reply:
@burgerkingeater The API is defined in tensorflow/tensorflow/c/experimental/filesystem/filesystem_interface.h though we add the implementation here. I have updated the comment of the code.